webextensions/api/proxy.json

```diff
@@ -498,7 +498,8 @@
               ],
               "firefox_android": [
                 {
-                  "version_added": "56"
+                  "version_added": "56",
+                  "version_removed": "79"
                 },
                 {
                   "alternative_name": "registerProxyScript",
```

Comment:
It struck me as… unlikely that the alternative_name would persist while the rest of the feature went away. Sure enough, `registerProxyScript` was removed in Firefox 71: https://bugzilla.mozilla.org/show_bug.cgi?id=1443259#c11

Can we close out this support statement for both Firefox desktop (71) and Android (79)?

Reply:
Thanks. For android this is version 79 that is first version after v71. I also updated the firefox desktop  case above. Needs to be checked.